dev/configs/.eslintrc.cjs

```diff
@@ -171,7 +185,7 @@ module.exports = defineConfig({
         {
             files: ["./dev/**"],
             rules: {
-                "no-restricted-imports": "off"
+                "@typescript-eslint/no-restricted-imports": "off"
```

Comment:
The context here is missing -- this change applies an override to the dev folder that turns off no-restricted-imports for dev generally.